sdk/monitor/azure-monitor-query/azure/monitor/query/_metrics_client.py

```diff
@@ -84,17 +84,17 @@ def query_resources(
         :keyword order_by: The aggregation to use for sorting results and the direction of the sort.
             Only one order can be specified. Examples: 'sum asc', 'maximum desc'.
         :paramtype order_by: Optional[str]
-        :keyword filter: The **$filter** is used to reduce the set of metric data returned. Example:
+        :keyword filter: The **filter** is used to reduce the set of metric data returned. Example:
             Metric contains metadata A, B and C. - Return all time series of C where A = a1 and B = b1 or
```

Comment:
It looks like the formatting of this paragraph was originally intended to be a bullet point list - can we restructure it to render like this?
<img width="298" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/8689453/272829be-5f2b-467a-9068-85acebbe227a">

Reply:
Might also be worth double-checking that the double-star bold tags are rendering correctly.

Reply:
Updated to improve bullet-point formatting.  And I can confirm that double-asterisks bold formatting works fine on both learn.microsoft.com and the sphinx docs site.